README.md

```diff
@@ -31,6 +36,16 @@ Contains the source code and documentation powering [clayui.com](https://clayui.
 ![Chrome Android](https://img.shields.io/badge/Chrome_Android-Latest-green.svg?style=flat)
 ![Safari iOS Latest](https://img.shields.io/badge/Safari_iOS-Latest-green.svg?style=flat)
 
+## Packages
+
+Clay repository follows the monorepo approach, all the major components that are the pillars of Clay are here in this repository.
+
+1. ### [Clay Css](./packages/clay-css): The pillar to give colors and structure to Clay, where is the css and examples of HTML markups of the components.
```

Comment:
```suggestion
1. **[Clay Css]**(./packages/clay-css): The pillar to give colors and structure to Clay, where is the css and examples of HTML markups of the components.
```
Reason: I don't think we need to use an `h2` here because it looks "big", bold should be enough